routes/recipes.js

```diff
@@ -6,7 +6,15 @@ module.exports = (function() {
   var express = require('express');
   var router = express.Router();
 
-  router.get('/', function(req, res, next){
+  function require_signin (req, res, next) {
```

Comment:
You should abstract this function out into a helper functions directory.

Reply:
ok will perform in the coming sprint